client/routes.js

```diff
@@ -22,12 +22,12 @@ class Routes extends Component {
         {/* Routes placed here are available to all visitors */}
         <Route path="/login" component={Login} />
         <Route path="/signup" component={Signup} />
+        <Route path="/create-form" component={FormBuilder} />
```

Comment:
I think this one slipped through 😄. We can move this back into the private routes with the `isLoggedIn` conditional

Reply:
Thanks for looking at it! It was working on the client side on my end- you may have to follow all the changes I made in some of the other files as well (it wasn't just entering in the secret key that made it work) I'd take a look at all the changes and try it again. 

I can send over an example of how to do the conditional in a second!